.github/workflows/tests.yml

```diff
@@ -50,6 +50,13 @@ jobs:
           DATABASE_URL: postgres://complaint_tracker:postgres@localhost:5432/complaint_tracker_test
         run: bundle exec rake spec
 
+      - name: Upload coverage information
```

Comment:
[question] Is the file / artifact being created at `coverage` coming out of the `spec` run step automatically? I'm assuming so, but I want to make sure I understand how that works!

Reply:
Yes, `bundle exec rake spec` runs the coverage stuff automatically